src/python/pants/backend/python/subsystems/python_tool_base.py

```diff
@@ -42,9 +42,10 @@ def register_options(cls, register):
             type=str,
             advanced=True,
             default=cls.default_entry_point,
-            help="The main module for the tool. If unspecified, the code using this tool "
-            "must provide it explicitly on invocation, or it can use the tool as a "
-            "library, invoked by a wrapper script.",
+            help=(
+                "The main module for the tool. Usually, you will not want to change this from the "
+                "default."
+            ),
```

Comment:
Idk why we expose an `--entry-point` option. I can't imagine wanting to override this. At that point, you should probably write a custom plugin, no?